loops-and-arrays.js

```diff
@@ -53,3 +59,6 @@ const squares = [30, 2, 8, 24, 11];
 //   'Het volume van 24 is 13824',
 //   'Het volume van 11 is 1331'
 // ]
+for (let i = 0; i < squares.length; i++) {
+    console.log("Het volume van " + squares[i] + " is " + squares[i]*squares[i]*squares[i])
+}
```

Comment:
Alles klopt en je hebt ook het netjes georganiseerd. Wel is het aan te raden om toch nog ; toe te voegen aan het einden van declaraties omdat dat toch bij het preferred code style hoort.